src/test/porter/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: porter
+description: Porter chart for KubeSphere
+type: application
+version: 0.1.1
+appVersion: 0.1.1
+icon: https://raw.githubusercontent.com/FeynmanZhou/landscape/master/hosted_logos/porter.svg
```

Comment:
@chinazj Please replace it with this URL: https://raw.githubusercontent.com/cncf/landscape/master/hosted_logos/porter.svg